### DISCUSSION TITLE
Active step as seen in other korg gear?

### DISCUSSION THREAD

**Discussion by ptowns** [bangcorrupt/hacktribe#159] [Ideas]
Do you think there might be a way to do active steps? I noticed you can turn off the trigger in the sequencer and still retain the pitch, velocity and gate data in the sequencer. Obviously with last step we can create poly rythems, but could we skip a step all together in the very korg way?
Also would be cool if trigger played what ever note was in the sequencer at theat time rather then c4... like default note is c4 unless there is other pitch data in the sequencer already.

**Comment by bangcorrupt** [comment]
I think we can do active step by latching step jump using [gate arp]+[touch pad] while holding the trigger pads.  It could be easier to use.  

A 'Seq Current'  trigger mode instead of just 'Seq 1st' is a good idea.

**Reply by Piergap** [reply]
that would be great indeed. adding flexibility in pattern playback!!! go!

**Comment by listenForward** [comment]
I like the suggestion of the idea of "Seq Current", but I'm not sure I fully understand the implication.

To my understanding, the behavior of the OG Pad-based Track Triggering will 
- start a given track when you hit the pad, 
- run them at the normal rate, at 16th steps.
- fire off events when when sequenced
- stop that track when releasing.

How would "Step Current" be different ?

**Reply by bangcorrupt** [reply]
Seq Current instead of Seq 1st, not Seq Play.  Play the note of  the current step instead of the first step.
